### PR TITLE
Move MID constants to a separate class

### PR DESCRIPTION
**Pull request by dstocco**
This allows a re-use of code in existing and future classes.

### CODE REVIEW

**Comment by dstocco** [comment]
Hi Matthias,
thanks for the feedback. I'm having a look, but AFAIU, I cannot define an inline function in the cxx if it's used in the cxx of other classes.
I think to understand that I have to define the inline function in the .h. I only have the choice between:
- declare it in the class body and define it outside
- define it in the class body

But in both cases I need to remove the .cxx and move everything in the .h.
The inline would indeed be more performant...so I can give it a try.

Thanks,
Cheers,
Diego

**Comment by dstocco** [comment]
Hi @matthiasrichter ,
I modified the code according to your suggestions, but the O2 compilation (or rather the tests) now fails. However, the failing test has nothing to do with my modifications:

`1/182 Test #179: o2sim_G4_mt ..........................................................***Failed  Required regular expression not found.Regex=[Macro finished succesfully
]  1.71 sec
[INFO] FairRootManager::OpenOutFile("o2sim.root")
SysError in <TFile::TFile>: could not delete o2sim.root (errno: 2) (No such file or directory)`

Cheers,
Diego

File: Detectors/MUON/MID/Base/include/MIDBase/Mapping.h [hunk 1]
```diff
@@ -86,10 +86,9 @@ class Mapping
   void buildDETypeMedium(int rpcType, std::array<int, 7> boards, bool largeNonBend);
   void buildDETypeCut(int rpcType, std::array<int, 7> boards, bool isBelowBeamPipe);
   void buildDETypeShort(int rpcType, std::array<int, 7> boards);
-  int getChamber(int deId) const;
-  int getRPCType(int deId) const;
+  /// Gets the RPC type of detection element deId
+  inline int getRPCType(int deId) const { return deId % 9; }
```
Comment (sawenzel) [inline Detectors/MUON/MID/Base/include/MIDBase/Mapping.h]: note that inline is only needed outside of the class body declaration. But it does not hurt (just mentioned it for the next time).

Comment (dstocco) [reply Detectors/MUON/MID/Base/include/MIDBase/Mapping.h]: HI @sawenzel ,
ok, thanks a lot for the suggestion: I'll take it into account for the next PR..
Cheers,
Diego

**Pull request merged**